pkg/detector/detector.go

```diff
@@ -271,11 +272,20 @@ func (d *ResourceDetector) EventFilter(obj interface{}) bool {
 		// The secret, with type 'kubernetes.io/service-account-token', is created along with `ServiceAccount` should be
 		// prevented from propagating.
 		// Refer to https://github.com/karmada-io/karmada/pull/1525#issuecomment-1091030659 for more details.
+		// The configmap and endpoints for leader election should be prevented from propagating.
```

Comment:
Shouldn't this comment be placed below? Other LGTM

Reply:
ok，thanks for reminding